### PR TITLE
docs(rxFeedback): add accidentally lost documentation

### PR DESCRIPTION
**Pull request by CITguy**
Add documentation lost in #1240.

### CODE REVIEW

**Comment by halkon** [comment]
homer-doh

File: src/rxFeedback/rxFeedback.js [hunk 1]
```diff
@@ -1,18 +1,129 @@
...
+
 /**
  * @ngdoc overview
  * @name rxFeedback
  * @description
  * # rxFeedback Component
  *
- * `rxFeedback` component gathers and sends user feedback to a default or specifiable email list.
- * 
+ * The rxFeedback component gathers and sends user feedback to a default or
```
Comment (halkon) [inline src/rxFeedback/rxFeedback.js]: why you no backtick? or at least bold it

Comment (CITguy) [inline src/rxFeedback/rxFeedback.js]: * backticks seem to denote that it is code (which it is not)
* I'm not sure bold is needed to emphasize the component considering the name of the component is mentioned at the top of the API docs page.

screen shot 2015-10-02 at 2 40 02 pm

Comment (halkon) [inline src/rxFeedback/rxFeedback.js]: Well, ... then


z1eiy

**Comment by halkon** [comment]
LGTM

**Pull request merged**